code/modules/mob/living/simple_animal/bot/secbot.dm

```diff
@@ -251,7 +252,10 @@ Auto Patrol: []"},
 	if(!C.handcuffed)
 		C.handcuffed = new /obj/item/restraints/handcuffs/cable/zipties/used(C)
 		C.update_handcuffed()
-		playsound(src, arrestsounds, 50, 0)//monkestation edit for custom arrest sounds
+		if(EMAGGED && prob(50)) //if it's emagged, there's a chance it'll play a special sound instead
+			playsound(src, emagsounds, 50, 0)
```

Comment:
This doesn't pick from the list, this is attempting to play the entire list as a sound.

Reply:
This doesn't seem to be the case for me.  While testing the latest changes I confirmed the hailer, beepsky attack sound, and beepsky arrest sound all pick randomly from their respective lists.